src/Sylius/Behat/Context/Api/Admin/ManagingProductsContext.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Admin;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Client\ApiClientInterface;
+use Webmozart\Assert\Assert;
+
+final class ManagingProductsContext implements Context
+{
+    /** @var ApiClientInterface */
+    private $client;
+
+    public function __construct(ApiClientInterface $client)
+    {
+        $this->client = $client;
+    }
+
+    /**
+     * @When I want to create a new configurable product
+     */
+    public function iWantToCreateANewConfigurableProduct(): void
+    {
+        $this->client->buildCreateRequest('products');
+    }
+
+    /**
+     * @When I specify its code as :code
+     * @When I do not specify its code
+     */
+    public function iSpecifyItsCodeAs(string $code = null): void
+    {
+        $this->client->addRequestData('code', $code);
+    }
+
+    /**
+     * @When I name it :name in :language
+     * @When I rename it to :name in :language
+     */
+    public function iRenameItToIn(string $name, string $language): void
+    {
+        $data = ['translations' => [$language => ['locale' => $language]]];
+        $data['translations'][$language]['name'] = $name;
```

Comment:
```suggestion
        $data = [
            'translations' => [
                $language => [
                    'locale' => $language,
                    'name' => $name,
                ],
            ],
        ];
```
```

Reply:
What will happen if `'locale' => $language` will be omitted?

Reply:
Not working